src/lib/UI/Config/ConfigWrapper.php

```diff
@@ -16,7 +16,7 @@ class ConfigWrapper implements \ArrayAccess, \JsonSerializable
     private $config;
 
     /**
-     * Config constructor.
+     * ConfigWrapper constructor.
```

Comment:
I know it's coming from PHPStorm but I suggest removing it as it doesn't provide any value.

Reply:
Fixed.